### PR TITLE
ISSUE-229 fix conversion factors and offsets (revealed with ABETCO)

### PR DESCRIPTION
**Pull request by jmkeil**
Fixes the wrong conversion factors and offsets noted in #229.

Please not that this PR is based on PR #269 to avoid later merge conflicts.

### CODE REVIEW

File: src/stateEnergyFlux.ttl [hunk 1]
```diff
@@ -115,8 +115,8 @@ sostef:XClass rdf:type owl:Class ;
 sostef:A1Class rdf:type owl:NamedIndividual ,
                      sostef:AClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "1.0E-8"^^xsd:double ;
-                                  sorelm:hasUpperBound "2.0E-8"^^xsd:double
+                                  sorelm:hasLowerBound "100000000.0"^^xsd:decimal ;
+                                  sorelm:hasUpperBound "200000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:119]: Shouldn't this be .00000002 instead of 20000000.0?

File: src/stateEnergyFlux.ttl [hunk 2]
```diff
@@ -115,8 +115,8 @@ sostef:XClass rdf:type owl:Class ;
 sostef:A1Class rdf:type owl:NamedIndividual ,
                      sostef:AClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "1.0E-8"^^xsd:double ;
-                                  sorelm:hasUpperBound "2.0E-8"^^xsd:double
+                                  sorelm:hasLowerBound "100000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:118]: Shouldn't this be .00000001 instead of 100000000.0?

File: src/stateEnergyFlux.ttl [hunk 3]
```diff
@@ -125,8 +125,8 @@ sostef:A1Class rdf:type owl:NamedIndividual ,
 sostef:A2Class rdf:type owl:NamedIndividual ,
                      sostef:AClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "2.0E-8"^^xsd:double ;
-                                  sorelm:hasUpperBound "3.0E-8"^^xsd:double
+                                  sorelm:hasLowerBound "200000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:128]: Shouldn't this be .00000002?

File: src/stateEnergyFlux.ttl [hunk 4]
```diff
@@ -125,8 +125,8 @@ sostef:A1Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "300000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:129]: Shouldn't this be .00000003?

File: src/stateEnergyFlux.ttl [hunk 5]
```diff
@@ -255,8 +255,8 @@ sostef:B5Class rdf:type owl:NamedIndividual ,
 sostef:B6Class rdf:type owl:NamedIndividual ,
                      sostef:BClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "6.0E-7"^^xsd:double ;
-                                  sorelm:hasUpperBound "7.0E-7"^^xsd:double
+                                  sorelm:hasLowerBound "60000000.0"^^xsd:decimal ;
+                                  sorelm:hasUpperBound "70000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:259]: sorelm:hasUpperBound ".0000007"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 6]
```diff
@@ -265,8 +265,8 @@ sostef:B6Class rdf:type owl:NamedIndividual ,
 sostef:B7Class rdf:type owl:NamedIndividual ,
                      sostef:BClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "7.0E-7"^^xsd:double ;
-                                  sorelm:hasUpperBound "8.0E-7"^^xsd:double
+                                  sorelm:hasLowerBound "70000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:268]: sorelm:hasLowerBound ".0000007"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 7]
```diff
@@ -265,8 +265,8 @@ sostef:B6Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "80000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:269]: sorelm:hasUpperBound ".0000008"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 8]
```diff
@@ -275,8 +275,8 @@ sostef:B7Class rdf:type owl:NamedIndividual ,
 sostef:B8Class rdf:type owl:NamedIndividual ,
                      sostef:BClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "8.0E-7"^^xsd:double ;
-                                  sorelm:hasUpperBound "9.0E-7"^^xsd:double
+                                  sorelm:hasLowerBound "80000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:278]: sorelm:hasLowerBound ".0000008"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 9]
```diff
@@ -275,8 +275,8 @@ sostef:B7Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "90000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:279]: sorelm:hasUpperBound ".0000009"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 10]
```diff
@@ -285,8 +285,8 @@ sostef:B8Class rdf:type owl:NamedIndividual ,
 sostef:B9Class rdf:type owl:NamedIndividual ,
                      sostef:BClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "9.0E-7"^^xsd:double ;
-                                  sorelm:hasUpperBound "1.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "90000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:288]: sorelm:hasLowerBound ".0000009"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 11]
```diff
@@ -285,8 +285,8 @@ sostef:B8Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "1000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:289]: sorelm:hasUpperBound ".000001"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 12]
```diff
@@ -295,8 +295,8 @@ sostef:B9Class rdf:type owl:NamedIndividual ,
 sostef:C1Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "1.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "2.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "1000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:298]: sorelm:hasLowerBound ".000001"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 13]
```diff
@@ -295,8 +295,8 @@ sostef:B9Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "2000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:299]: sorelm:hasUpperBound ".0000002"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 14]
```diff
@@ -305,8 +305,8 @@ sostef:C1Class rdf:type owl:NamedIndividual ,
 sostef:C2Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "2.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "3.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "2000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:308]: sorelm:hasLowerBound ".000002"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 15]
```diff
@@ -305,8 +305,8 @@ sostef:C1Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "3000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:309]: sorelm:hasUpperBound ".000003"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 16]
```diff
@@ -315,8 +315,8 @@ sostef:C2Class rdf:type owl:NamedIndividual ,
 sostef:C3Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "3.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "4.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "3000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:318]: sorelm:hasLowerBound ".000003"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 17]
```diff
@@ -325,8 +325,8 @@ sostef:C3Class rdf:type owl:NamedIndividual ,
 sostef:C4Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "4.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "5.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "4000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:328]: sorelm:hasLowerBound ".000004"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 18]
```diff
@@ -325,8 +325,8 @@ sostef:C3Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "5000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:329]: sorelm:hasUpperBound ".000005"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 19]
```diff
@@ -335,8 +335,8 @@ sostef:C4Class rdf:type owl:NamedIndividual ,
 sostef:C5Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "5.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "6.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "5000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:338]: sorelm:hasLowerBound ".000005"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 20]
```diff
@@ -335,8 +335,8 @@ sostef:C4Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "6000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:339]: sorelm:hasUpperBound ".000006"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 21]
```diff
@@ -345,8 +345,8 @@ sostef:C5Class rdf:type owl:NamedIndividual ,
 sostef:C6Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "6.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "7.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "6000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:348]: sorelm:hasLowerBound ".000006"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 22]
```diff
@@ -345,8 +345,8 @@ sostef:C5Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "7000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:349]: sorelm:hasUpperBound ".000007"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 23]
```diff
@@ -355,8 +355,8 @@ sostef:C6Class rdf:type owl:NamedIndividual ,
 sostef:C7Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "7.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "8.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "7000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:358]: sorelm:hasLowerBound ".000007"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 24]
```diff
@@ -355,8 +355,8 @@ sostef:C6Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "8000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:359]: sorelm:hasUpperBound ".000008"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 25]
```diff
@@ -365,8 +365,8 @@ sostef:C7Class rdf:type owl:NamedIndividual ,
 sostef:C8Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "8.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "9.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "8000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:368]: sorelm:hasLowerBound ".000008"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 26]
```diff
@@ -365,8 +365,8 @@ sostef:C7Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "9000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:369]: sorelm:hasUpperBound ".000009"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 27]
```diff
@@ -375,8 +375,8 @@ sostef:C8Class rdf:type owl:NamedIndividual ,
 sostef:C9Class rdf:type owl:NamedIndividual ,
                      sostef:CClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "9.0E-6"^^xsd:double ;
-                                  sorelm:hasUpperBound "1.0E-6"^^xsd:double
+                                  sorelm:hasLowerBound "9000000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:378]: sorelm:hasLowerBound ".000009"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 28]
```diff
@@ -375,8 +375,8 @@ sostef:C8Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "1000000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:379]: sorelm:hasUpperBound ".000001"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 29]
```diff
@@ -385,8 +385,8 @@ sostef:C9Class rdf:type owl:NamedIndividual ,
 sostef:M1Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "1.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "2.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "100000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:388]: sorelm:hasLowerBound ".00001"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 30]
```diff
@@ -385,8 +385,8 @@ sostef:C9Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "200000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:389]: sorelm:hasUpperBound ".00002"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 31]
```diff
@@ -395,8 +395,8 @@ sostef:M1Class rdf:type owl:NamedIndividual ,
 sostef:M2Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "2.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "3.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "200000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:398]: sorelm:hasLowerBound ".00002"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 32]
```diff
@@ -395,8 +395,8 @@ sostef:M1Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "300000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:399]: sorelm:hasUpperBound ".00003"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 33]
```diff
@@ -405,8 +405,8 @@ sostef:M2Class rdf:type owl:NamedIndividual ,
 sostef:M3Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "3.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "4.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "300000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:408]: sorelm:hasLowerBound ".00003"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 34]
```diff
@@ -405,8 +405,8 @@ sostef:M2Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "400000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:409]: sorelm:hasUpperBound ".00004"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 35]
```diff
@@ -415,8 +415,8 @@ sostef:M3Class rdf:type owl:NamedIndividual ,
 sostef:M4Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "4.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "5.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "400000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:418]: sorelm:hasLowerBound ".00004"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 36]
```diff
@@ -415,8 +415,8 @@ sostef:M3Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "500000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:419]: sorelm:hasUpperBound ".00005"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 37]
```diff
@@ -425,8 +425,8 @@ sostef:M4Class rdf:type owl:NamedIndividual ,
 sostef:M5Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "5.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "6.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "500000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:428]: sorelm:hasLowerBound ".00005"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 38]
```diff
@@ -425,8 +425,8 @@ sostef:M4Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "600000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:429]: sorelm:hasUpperBound ".00006"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 39]
```diff
@@ -435,8 +435,8 @@ sostef:M5Class rdf:type owl:NamedIndividual ,
 sostef:M6Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "6.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "7.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "600000.0"^^xsd:decimal ;
+                                  sorelm:hasUpperBound "700000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:439]: sorelm:hasUpperBound ".00007"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 40]
```diff
@@ -445,8 +445,8 @@ sostef:M6Class rdf:type owl:NamedIndividual ,
 sostef:M7Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "7.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "8.0E-5"^^xsd:double
+                                  sorelm:hasLowerBound "700000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:448]: sorelm:hasLowerBound ".00007"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 41]
```diff
@@ -445,8 +445,8 @@ sostef:M6Class rdf:type owl:NamedIndividual ,
...
+                                  sorelm:hasUpperBound "800000.0"^^xsd:decimal
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:449]: sorelm:hasUpperBound ".00008"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 42]
```diff
@@ -465,8 +465,8 @@ sostef:M8Class rdf:type owl:NamedIndividual ,
 sostef:M9Class rdf:type owl:NamedIndividual ,
                      sostef:MClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "9.0E-5"^^xsd:double ;
-                                  sorelm:hasUpperBound "1.0E-4"^^xsd:double
+                                  sorelm:hasLowerBound "900000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:468]: sorelm:hasLowerBound ".00009"^^xsd:decimal ;

File: src/stateEnergyFlux.ttl [hunk 43]
```diff
@@ -915,8 +915,8 @@ sostef:X4Class rdf:type owl:NamedIndividual ,
 sostef:X5Class rdf:type owl:NamedIndividual ,
                      sostef:XClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "5.0E-4"^^xsd:double ;
-                                  sorelm:hasUpperBound "6.0E-4"^^xsd:double
+                                  sorelm:hasLowerBound "50000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:918]: sorelm:hasLowerBound ".0005"^^xsd:decimal ;
                                  sorelm:hasUpperBound ".0006"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 44]
```diff
@@ -925,8 +925,8 @@ sostef:X5Class rdf:type owl:NamedIndividual ,
 sostef:X6Class rdf:type owl:NamedIndividual ,
                      sostef:XClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "6.0E-4"^^xsd:double ;
-                                  sorelm:hasUpperBound "7.0E-4"^^xsd:double
+                                  sorelm:hasLowerBound "60000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:928]: sorelm:hasLowerBound ".0006"^^xsd:decimal ;
                                  sorelm:hasUpperBound ".0007"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 45]
```diff
@@ -945,8 +945,8 @@ sostef:X7Class rdf:type owl:NamedIndividual ,
 sostef:X8Class rdf:type owl:NamedIndividual ,
                      sostef:XClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "8.0E-4"^^xsd:double ;
-                                  sorelm:hasUpperBound "9.0E-4"^^xsd:double
+                                  sorelm:hasLowerBound "80000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:948]: sorelm:hasLowerBound ".0008"^^xsd:decimal ;
                                  sorelm:hasUpperBound ".0009"^^xsd:decimal

File: src/stateEnergyFlux.ttl [hunk 46]
```diff
@@ -955,8 +955,8 @@ sostef:X8Class rdf:type owl:NamedIndividual ,
 sostef:X9Class rdf:type owl:NamedIndividual ,
                      sostef:XClass ;
             sorelph:hasEnergyFlux [ rdf:type sopropef:EnergyFlux_Wm2 ;
-                                  sorelm:hasLowerBound "9.0E-4"^^xsd:double ;
-                                  sorelm:hasUpperBound "0.001"^^xsd:double
+                                  sorelm:hasLowerBound "90000.0"^^xsd:decimal ;
```
Comment (rduerr) [inline src/stateEnergyFlux.ttl:958]: sorelm:hasLowerBound ".0009"^^xsd:decimal ;

**Review by rduerr — CHANGES_REQUESTED** [review]
The stateEnergyFlux file has a large number of issues where numbers like 4.0e-3 got converted into something like 4000 instead of .004.  These all have comments and most have a line that could be cut and pasted to fix the problem.

**Review by rduerr — CHANGES_REQUESTED** [review]
The stateEnergyFlux file has a large number of issues where numbers like 4.0e-3 got converted into something like 4000 instead of .004.  These all have comments and most have a line that could be cut and pasted to fix the problem.

**Comment by brandonnodnarb** [comment]
addressed in #269

**Pull request closed without merging**